### PR TITLE
Deploying to AWS Lambda documentation

### PR DESCRIPTION
**Pull request by nikolalsvk**

### CODE REVIEW

**Comment by shiroyasha** [comment]
Nice :+1: 

The only this I would consider is putting a larger padding around your screenshots. On GitHub it looks a little "squished".

**Comment by nikolalsvk** [comment]
@shiroyasha Hmm, can you give me a hint on how to do that? I think it will look nicer on Semaphore docs site because of the classes `img-responsive img-bordered`.

**Comment by shiroyasha** [comment]
If it looks OK on semaphore, than it is ok :+1:

File: source/docs/deploying-to-aws-lambda.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Deploy to AWS Lambda
+category: Deploying your application
+---
+
+Amazon Web Services (AWS) is a set of cloud services that help you manage your
+web applications, servers, databases and more. AWS Lambda is a compute service
+where you can upload your code to Lambda and the service can run the code
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: to which you can upload your code, and then the service can run the code

File: source/docs/deploying-to-aws-lambda.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+on your behalf using AWS infrastructure.
+
+Semaphore has a build-in integration, which makes continuous deployment to AWS
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: has a build-in integration, which -> has a built-in integration which

File: source/docs/deploying-to-aws-lambda.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Lambda very easy. This guide shows you how you can set it up and assumes that
+you have already an application that is configured on AWS Lambda.
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: you have already an application -> you already have an application

File: source/docs/deploying-to-aws-lambda.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## Choosing the deployment strategy
+
+There are two deployment strategies on Semaphore: "automatic" and "manual".
+
+**Automatic** means that a deploy will be triggered after every passed build on
+the selected branch. In addition, you can also manually deploy any build from
+any branch at any time.
+
+The **Manual** strategy requires manual selection of the builds to deploy.
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: requires manual selection -> requires the manual selection

File: source/docs/deploying-to-aws-lambda.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+For automatic deployment, you will be asked to select which branch will be
+automatically deployed after each passed build.
+
+**Note**: You can easily change the deployment strategy from server settings at
+any time after the setup is complete.
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: after the setup is complete -> once the setup is complete

File: source/docs/deploying-to-aws-lambda.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/223/Screen_Shot_2015-12-11_at_9.24.29_AM.png"
+alt="AWS Lambda choose branch" class="img-responsive img-bordered">
+
+## Entering the AWS credentials
+
+On the following screen, enter your AWS credentials. If you need help getting
+these credentials, consult the "Retrieving Security Credentials" section of
+Semaphore's <a href="https://semaphoreci.com/community/tutorials/how-to-deploy-a-ruby-on-rails-application-to-elastic-beanstalk-with-semaphore" alt="How to Deploy a Ruby on Rails Application to Elastic Beanstalk with Semaphore">
+tutorial on deploying a web application on Elastic Beanstalk</a>. Process for
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: Process -> The process

File: source/docs/deploying-to-aws-lambda.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+retrieving credentials is the same when deploying to Lambda.
+
+After entering the credentials, select the region your application resides in
+on the same screen. This will enable Semaphore to list functions you have in
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: list functions you have in -> list the functions in

File: source/docs/deploying-to-aws-lambda.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+the region you specified.
+
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/224/Screen_Shot_2015-12-11_at_9.24.55_AM.png"
+alt="Enter AWS Credentials" class="img-responsive img-bordered">
+
+## Selecting the function and typing in dependencies installation commands
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: Erase "and typing in dependencies installation commands"

File: source/docs/deploying-to-aws-lambda.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+Next, you can choose to which Lambda function you want to deploy your code.
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: you can choose to which Lambda function you want to deploy your code -> you can choose the Lambda function to which you wish to deploy your code

File: source/docs/deploying-to-aws-lambda.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+Also, there are suggested commands for installing dependencies in your project.
+This is needed because Lambda needs all source files of the packages you use
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: all source files -> all the source files

File: source/docs/deploying-to-aws-lambda.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+In this step, you need to provide a name for your server, which can be any name
+of your choice. It will be used on your Semaphore dashboard and deployment
+timeline. During this step, you may also choose to set a server URL for quick
+access on Semaphore.
+
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/226/Screen_Shot_2015-12-11_at_9.26.02_AM.png"
+alt="Naming your server" class="img-responsive img-bordered">
+
+The setup is now complete - you are ready to automatically deploy to Lambda.
+All you need to do is click "Deploy".
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: click "Deploy" -> click on _Deploy_

File: source/docs/deploying-to-aws-lambda.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+## Selecting the function and typing in dependencies installation commands
+
+Next, you can choose to which Lambda function you want to deploy your code.
+Also, there are suggested commands for installing dependencies in your project.
+This is needed because Lambda needs all source files of the packages you use
+in your project.
+
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/225/Screen_Shot_2015-12-11_at_9.25.29_AM.png"
+alt="Choose Lambda function" class="img-responsive img-bordered">
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: Choose Lambda function -> Selecting the Lambda function

File: source/docs/deploying-to-aws-lambda.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/226/Screen_Shot_2015-12-11_at_9.26.02_AM.png"
+alt="Naming your server" class="img-responsive img-bordered">
+
+The setup is now complete - you are ready to automatically deploy to Lambda.
+All you need to do is click "Deploy".
+
+<img src="https://d2l3jyjp24noqc.cloudfront.net/uploads/image/img/227/Screen_Shot_2015-12-11_at_9.26.14_AM.png"
+alt="Ready to deploy" class="img-responsive img-bordered">
+
+Happy building!
```
Comment (dunjardl) [inline source/docs/deploying-to-aws-lambda.md]: Remove the "Happy building!" part :)

**Pull request merged**